web/views/_nav.erb

```diff
@@ -36,13 +36,6 @@
           <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-expanded="false">
```

Comment:
If you remove the dropdown-menu, why keep the dropdown-toggle?

This dropdown was added so that smaller desktops (like iPads) wouldn't see the tabs overflow the right and break onto a second line of tabs.  This doesn't happen when you have "Sidekiq Pro" and only one extra tab "Batches", but it does happen when you have "Sidekiq Enterprise" and three extra tabs "Batches", "Limits" and "Cron".  Unfortunately it's tough to get a design that works well with all three packages.

Reply:
Ah true, you are right. I was't 100% why that was there but that makes sense. I went and removed it. I think seeing the mobile nav on iPads should be okay.